html_ui/Pages/VCockpit/Instruments/Airliners/B747_8/EICAS/Pages/B747_8_LowerEICAS_GEAR.html

```diff
@@ -0,0 +1,147 @@
+<link rel="stylesheet" href="./B747_8_LowerEICAS_GEAR.css" />
+
+<script type="text/html" id="B747_8LowerEICASGEARTemplate">
+    <svg viewBox="0 0 600 600" xmlns="http://www.w3.org/2000/svg" style="background:black;">
+        <g id="fixed-elements">
+  					<ellipse ry="5" rx="5" cy="86" cx="300" />
+  					<rect rx="15" height="49" width="30" y="60" x="259" />
+  					<rect rx="15" height="49" width="30" y="60" x="311" />
+  					<line y2="86" x2="295" y1="86" x1="290" />
+  					<rect height="21" width="65" y="117" x="268" />
+  					<rect rx="15" height="47" width="30" y="250" x="107" />
+ 					 	<line y2="86" x2="311" y1="86" x1="305" />
+  					<rect rx="15" height="47" width="30" y="302" x="107" />
+  					<rect rx="15" height="47" width="30" y="250" x="153" />
+  					<rect rx="15" height="47" width="30" y="302" x="153" />
+						<line y2="273" x2="152" y1="273" x1="137" />
+  					<line y2="327" x2="153" y1="327" x1="138" />
+  					<ellipse ry="5" rx="5" cy="300" cx="145" />
+  					<line y2="294" x2="145" y1="273" x1="145" />
+  					<line y2="327" x2="145" y1="305" x1="145" />
+  					<rect rx="15" height="47" width="30" y="250" x="417" />
+  					<rect rx="15" height="47" width="30" y="302" x="417" />
+  					<rect rx="15" height="47" width="30" y="250" x="463" />
+  					<rect rx="15" height="47" width="30" y="302" x="463" />
+						<line y2="273" x2="463" y1="273" x1="448" />
+  					<line y2="327" x2="463" y1="327" x1="448" />
+  					<ellipse ry="5" rx="5" cy="300" cx="455" />
+  					<line y2="294" x2="455" y1="273" x1="455" />
+  					<line y2="327" x2="455" y1="305" x1="455" />
+  					<rect rx="15" height="47" width="30" y="404" x="393" />
+ 					 	<rect rx="15" height="47" width="30" y="456" x="393" />
```

Comment:
Please fix formatting (whole file)

Reply:
Formatting has been fixed (And apparently I need a new text editor...)